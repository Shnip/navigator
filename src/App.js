import React, { Component } from 'react';
import Routes from './components/Route/Routes';
import {Header} from './components/Header/Header';
import {Footer} from './components/Footer/Footer';
import './sass/index.sass';


class App extends Component {
  render() {
    return (
      <div className="container">
        <Header />
        <Routes />
        <Footer />
      </div>
    );
  }
}

export default App;
