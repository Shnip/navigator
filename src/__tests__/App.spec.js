import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

describe('test App component', () => {
  const component = shallow(<App />);

  it ('renders Header component', () => {
    expect(component.find('Header')).toHaveLength(1);
  });

  it ('renders Routes component', () => {
    expect(component.find('Routes')).toHaveLength(1);
  });

  it ('renders Footer component', () => {
    expect(component.find('Footer')).toHaveLength(1);
  });

  it ('renders properly', () => {
    expect(component).toMatchSnapshot();
  });
})
