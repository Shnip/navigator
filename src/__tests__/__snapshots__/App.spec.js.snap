// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test App component renders properly 1`] = `ShallowWrapper {}`;
